src/pages/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Weather from '../components/Weather';
+
+const Home = () => (
```

Comment:
- Great work creating a home page. However, it seems your homepage has not met the project guidelines. Kindly check below ⬇️ 

**Filtering**

- [ ]  It will be great to ensure that the list of categories that are displayed on your homepage can be filtered with some value. Example category name. In your application, you have created a list of 4 coordinates as your categories but there is no way to filter them. Hence this project requirement is not met. Kindly ensure there is a way to filter the categories in your application.

**Numeric values**

- [ ] It will be great to ensure that there are some numeric values of each category displayed.  These numeric values should come from the API. For instance, in your implementation, you can decide to show some of the concentration levels. 

![Screenshot 2023-05-27 at 12 00 04](https://github.com/kalbek/react-capstone/assets/32361963/405cf07e-eec3-4990-b1b2-92272e88ba66)

Kindly you can confirm the project requirements on this [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)